internal/cmd/types/file.go

```diff
@@ -0,0 +1,38 @@
+package types
+
+import (
+	"io"
+	"os"
+)
+
+// StringOrFile is a pflag.Value type that can be used to read a value either
+// from the command line flag, or as path to a file.
+// If the value is not an existing filepath, it will be used as the literal
```

Comment:
The idea here is to avoid having separate flags for a literal cert or a file which contains the cert.

Calling code should only ever see the final value:
1. `--tls-trusted-cert=cert.pem`, `options.TrustedCertificate` will have the contents of `cert.pem`
2. `--tls-trusted-cert="-----BEGIN CERTIFICATE----...."`,  `options.TrustedCertificate` will have the PEM encoded cert
3. `--tls-trusted-cert=non-existant-file.pem`, error "Failed to read trusted certificates for server"